.prettierrc.js

```diff
@@ -2,4 +2,12 @@
 
 module.exports = {
   singleQuote: true,
+  overrides: [
+    {
+      files: '*.hbs',
+      options: {
+        singleQuote: false,
```

Comment:
This allows us to use prettier for handlebars files without it changing all quotes to `'`.